app.rb

```diff
@@ -0,0 +1,70 @@
+require_relative 'bubble'
+require "colorize"
+
+
+File.open("/Users/Chris/Code/class_projects/soapbox/soapbox/.soapbox", "w") do |file|
```

Comment:
Comment for the future, all of this user login stuff is a bit scattered (some here, some in `bubble.rb`), spending some time to move it all into one place would be good. It would be best in it's own class or something, but even just methods like you did below with `entire_feed`.